__tests__/app.test.js

```diff
@@ -132,3 +133,41 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/article_id/comments", () => {
+  test("POST: 201 Adds a comment for an article", () => {
+    const userObj = { username: "butter_bridge", body: "This is a comment!" };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(userObj)
+      .expect(201)
+      .then((response) => {
+        expect(typeof response.body.comment.comment_id).toBe("number");
+        expect(typeof response.body.comment.article_id).toBe("number");
+        expect(response.body.comment.author).toBe("butter_bridge");
+        expect(response.body.comment.body).toBe("This is a comment!");
+        expect(typeof response.body.comment.created_at).toBe("string");
+        expect(typeof response.body.comment.votes).toBe("number");
+      });
+  });
+  test("POST: 400 Returns a bad request message if no user is provided", () => {
+    const userObj = { username: "", body: "This is a comment!" };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(userObj)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Empty user!");
+      });
+  });
+  test("POST: 400 Returns a bad request message if no body is provided", () => {
+    const userObj = { username: "butter_bridge", body: "" };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(userObj)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Empty body!");
+      });
+  });
```

Comment:
Excellent coverage so far!

I've also like to see a couple more tests: 

-if the username isn't in the database
-if the article_id is invalid
-if the article_id does not exist